.github/workflows/buildAndTest.yml

```diff
@@ -66,3 +66,9 @@ jobs:
       shell: bash
       run: |
         ./build_tools/github_actions/ci_build_stablehlo_python_api.sh "$LLVM_PROJECT_DIR" "$STABLEHLO_PYTHON_BUILD_DIR" "$GITHUB_WORKSPACE"
+
+    # Run Bazel for builds
+    - name: Build and Test StableHLO using Bazel
```

Comment:
We can leave this here for now. I may suggest we separate this into another action since these steps are run serially. This means we're probably adding +1hr to our llvm revision bump jobs

Reply:
Let's see how long it takes before we make that decision though.

Reply:
It takes around 55 min to run the Bazel without cache, adding to the ~4hr of the LLVM integrate. I've created a separate workflow for it since this is a 20% increase in build time.